OpenDocumentReader/DocumentViewController.swift

```diff
@@ -83,6 +88,32 @@ class DocumentViewController: UIViewController, DocumentDelegate, GADBannerViewD
         loadBannerAd()
     }
     
+    func setVCcontraints() {
```

Comment:
can't we configure this via the Xcode UI?

Reply:
Of course it is possible, but more complicated to implement any changes in the future, if some problem like this occurs. Interface Builder has too many different options in different places. I would say, constraints via code are much easier to understand.

Reply:
But let me know if you still want to have all the constraints via IB.